cli/main.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/Microsoft/cobalt/cli/cmd"
+	"github.com/microsoft/cobalt/cli/cmd"
```

Comment:
Nit: Good practice to be case-sensitive. 
```suggestion
	"github.com/Microsoft/cobalt/cli/cmd"
```

Reply:
@iphilpot Microsoft actually just changed its org repo to lowercase "m" a few weeks ago. 
 - https://twitter.com/mschoening/status/1125521372828934144
 -  https://twitter.com/martinwoodward/status/1125858101075357699